### DISCUSSION TITLE
No content sent to Obsidian (Edge)

### DISCUSSION THREAD

**Discussion by denny0810** [coddingtonbear/obsidian-web#163] [Bug Report]
I received this warning as opening the options. all note recall features are disabled. After clipping the webpage, file was created but the content was empty, seeming nothing was captured. Edge browser is v121.0.2277.112 and the obsidian verison is v1.5.3 . I attached the bug report here.  
[obsidian-local-rest-api.bug_report.json](https://github.com/coddingtonbear/obsidian-web/files/14231750/obsidian-local-rest-api.bug_report.json)
**thanks.**

**Comment by coddingtonbear** [comment]
I'm afraid that message is accurate!  I do not test this extension in any environment other than Chrome, and Edge is not Chrome.   Things *may* work correctly there, but if they do, it's an accident!

Maybe somebody else who's using Edge can help you, though.

**Reply by denny0810** [reply]
I appoligized, the information is not so clear. The extension can work now though the warning is till there.  I may set an unfit configure that caused the empty output, which I'm still looking for the reason.

**Comment by coddingtonbear** [comment]
This turns out to be a real bug caused by some accidental timing changes around when page context was gathered.  I've updated the project and sent a new version to the Chrome App Store for their review.  It's difficult to say how long their review process might take, but the most recent version was approved for release within a few hours.

**Comment by coddingtonbear** [comment]
The fix for this has been approved by the app store.  Your browser will automatically fetch new extension versions on its own, but you can ask it to download the update immediately by following the instructions here: https://github.com/coddingtonbear/obsidian-web/wiki/How-do-I-ask-Chrome-to-update-extensions-immediately%3F
